src/client/client.ts

```diff
@@ -354,10 +363,17 @@ export class Client {
    * ```
    */
   public async events(req?: EventsRequest): Promise<GetEventsResponse> {
-    const query: Record<string, number> = {};
-    if (req?.cursor && req.cursor > 0) {
-      query.cursor = req.cursor;
-    }
+    const query: Record<string, string> = {};
+    Object.keys(req ?? {}).forEach(key => {
```

Comment:
We could use `Object.entries`  here, to both get the key and value

Reply:
Ah yes, ty